File: despatch.Domain/Models/Default/Response/UserImageResponse.cs
namespace despatch.Domain.Models.Default.Response
{
    public class UserImageResponse : BaseEntityResponse
    {
        public string UserId { get; set; }
        public string ImageId { get; set; }

        public virtual UserResponse User { get; set; }
        public virtual ImageResponse Image { get; set; }
    }
}

File: despatch.WEB/app/src/swagger/models/user-image-response.ts
/* tslint:disable */
import { BaseEntityResponse } from './base-entity-response';
import { ImageResponse } from './image-response';
import { UserResponse } from './user-response';
export interface UserImageResponse extends BaseEntityResponse{
  image?: ImageResponse;
  imageId?: string;
  user?: UserResponse;
  userId?: string;
}
